roles/foreman/tasks/main.yaml

```diff
@@ -76,7 +76,7 @@
     network: host
     hostname: "{{ ansible_fqdn }}"
     secrets:
-      - 'foreman-database-url,type=env,target=DATABASE_URL'
+      - 'foreman-database-yaml,type=mount,target=/etc/foreman/database.yml'
```

Comment:
the change from env to a file is because you want to be able to set ssl mode etc?

Reply:
That and I couldn't see a way to set DB pools without this file.

Reply:
https://guides.rubyonrails.org/configuring.html#configuring-a-database says you add `?pool=5` to the URL.